### PR TITLE
Fix/119630 hide unavailable tickets

### PR DESCRIPTION
**Pull request by paulmskim**
For ticket: https://central.tri.be/issues/ 119630

Note: changed `blockId` to `clientId` and selector names for consistency, readability, and ease of understanding.

### CODE REVIEW

**Comment by tr1b0t** [comment]
Closing this Pull Request because it is pull requested against the wrong branch. The Central ticket is for *G19.01*, but this was pull requested against *release/M19.01*.

File: src/views/blocks/tickets.php [hunk 1]
```diff
@@ -20,10 +20,30 @@
 $cart_url     = $this->get( 'cart_url' );
 $cart_classes = array( 'tribe-block', 'tribe-block__tickets' );
 
-// We don't display anything if there is not provider
-if ( ! $provider ) {
+// We don't display anything if there is no provider or tickets
+if ( ! $provider || empty( count( $tickets ) ) ) {
```
Comment (sc0ttkclark) [inline src/views/blocks/tickets.php]: Using empty() like this is PHP 5.5+ only (empty only supports variables prior to 5.5)

File: src/views/blocks/tickets/content-inactive.php [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+ *
+ * See more documentation about our Blocks Editor templating system.
+ *
+ * @link {INSERT_ARTCILE_LINK_HERE}
+ *
+ * @version 4.9
+ *
+ */
+
+$message = $this->get( 'sale_past' ) ? esc_html__( 'Tickets are no longer available', 'event-tickets' ) : esc_html__( 'Tickets are not yet available', 'event-tickets' );
```
Comment (sc0ttkclark) [inline src/views/blocks/tickets/content-inactive.php]: Use __()

File: src/views/blocks/tickets/content-inactive.php [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+?>
+<div
+	class="tribe-block__tickets__item__content tribe-block__tickets__item__content--inactive"
+>
+	<?php echo $message ?>
```
Comment (sc0ttkclark) [inline src/views/blocks/tickets/content-inactive.php]: Use esc_html for $message for late escaping

**Pull request merged**